src/main/java/pl/touk/controller/DriverController.java

```diff
@@ -0,0 +1,31 @@
+package pl.touk.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import pl.touk.service.DriverService;
+import pl.touk.service.dto.DriverDto;
+import pl.touk.service.dto.NewDriverDto;
+import pl.touk.service.dto.UpdateDriverDto;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class DriverController {
+
+    private final DriverService driverService;
+
+    @PostMapping("/newDriver")
```

Comment:
czy to jest ścieżka zgodna z RESTem?

Reply:
Nie jest. Powinienem dodać URI dla kontrollera @RequestMapping("/driver") a następnie nad metodą adnotacje @PostMapping("/new").

Reply:
A czy `/new` jest potrzebne jak to zrobić inaczej?

Reply:
/new nie jest wymagane, możemy je pominąć. Poprzez konwencje nazewnictwa i metodę wiemy co zwróci dany request. Jednak moim zdaniem dobrze jest zostawić /new, ułatwi debugowanie.